astro_2.0/components/DaoDetails/DaoDetailsMinimized.module.scss

```diff
@@ -42,9 +42,15 @@
 
 .controls {
   align-items: center;
+  column-gap: 2px;
```

Comment:
We might have issues with this prop in Safari. Can you check if it works there, please?

Reply:
Don't have Safari, replaced `column-gap` with `gap`.